source/ToSort/XSharp.Playground/XSharp.Playground.xsproj

```diff
@@ -1,15 +1,5 @@
-﻿<Project>
-
-    <Import Project="..\XSharp.Sdk\obj\Debug\package\Sdk\Sdk.props" />
-
-    <PropertyGroup>
-        <OutputType>Bootable</OutputType>
-    </PropertyGroup>
-
-    <PropertyGroup>
-        <XSharpDesignTimeTargetsPath>$(LocalAppData)\CustomProjectSystems\XSharp\XSharp.DesignTime.targets</XSharpDesignTimeTargetsPath>
-    </PropertyGroup>
-    
-    <Import Project="..\XSharp.Sdk\obj\Debug\package\Sdk\Sdk.targets" />
-
-</Project>
+﻿<Project Sdk="XSharp.Sdk" ToolsVersion="Current">
```

Comment:
Remove `ToolsVersion="Current"`.

Reply:
Also, if you want to keep this project as a local test project (i.e. to try local changes), you can revert the changes and change the `XSharpDesignTimeTargetsPath` property to `..\XSharp.ProjectSystem\BuildSystem\XSharp.DesignTime.targets`. If you want to update `XSharp.Sdk` version, check `global.json`.